Packs/CommonScripts/Scripts/IsIPPrivate/IsIPPrivate.py

```diff
@@ -0,0 +1,116 @@
+from ipaddress import IPv4Address, IPv4Network
+from time import sleep
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+DELIMETER = ","
```

Comment:
I think we can remove this global variable and just use "," as it is shorter and probably will not change in the future

Reply:
I prefer to keep it in case we decide to later make changes in how the IP list is structured. E.g., if we decide to later have a JSON with a "tag" property, instead of having the tag come after the delimeter